src/RunTime.cpp

```diff
@@ -0,0 +1,455 @@
+      int i = 0;
+      arg_index++;
+      char *buf = argv[arg_index];
+      char *outer_ptr = NULL;
+      char *p;
+      while((p = strtok_r(buf, ",", &outer_ptr)) != NULL) {
+        *pointer[i] = atoi(p);
+        i++;
+        buf = NULL;
+      }
+      arg_index++;
+    }
+    else if(strcmp(argv[arg_index], "-CMFD_lattice") == 0) {
+      int *pointer[] = {&RP._NCx, &RP._NCy, &RP._NCz};
+      int i = 0;
+      arg_index++;
+      char *buf = argv[arg_index];
+      char *outer_ptr = NULL;
+      char *p;
+      while((p = strtok_r(buf, ",", &outer_ptr)) != NULL) {
+        *pointer[i] = atoi(p);
+        i++;
+        buf = NULL;
+      }
+      arg_index++;
+    }
+    else if(strcmp(argv[arg_index], "-output_mesh_lattice") == 0) {
+      std::vector<int> tmp;
+      arg_index++;
+      char *buf = argv[arg_index];
+      char *outer_ptr = NULL;
+      char *p;
+      while((p = strtok_r(buf, ",", &outer_ptr)) != NULL) {
+        tmp.push_back(atoi(p));
+        buf = NULL;
+      }
+      RP._output_mesh_lattices.push_back(tmp);
+      arg_index++;
+    }
+    else if(strcmp(argv[arg_index], "-output_type") == 0 ) {
+      arg_index++;
+      RP._output_types.push_back(atoi(argv[arg_index++]));
+    }
+    else if(strcmp(argv[arg_index], "-num_threads") == 0 ) {
+      arg_index++;
+      RP._num_threads = atoi(argv[arg_index++]);
+    }
+    else if(strcmp(argv[arg_index], "-azim_spacing") == 0) {
+      arg_index++;
+      RP._azim_spacing = atof(argv[arg_index++]);
+    }
+    else if(strcmp(argv[arg_index], "-num_azim") == 0) {
+      arg_index++;
+      RP._num_azim = atoi(argv[arg_index++]);
+    }
+    else if(strcmp(argv[arg_index], "-polar_spacing") == 0) {
+      arg_index++;
+      RP._polar_spacing = atof(argv[arg_index++]);
+    }
+    else if(strcmp(argv[arg_index], "-num_polar") == 0) {
+      arg_index++;
+      RP._num_polar = atoi(argv[arg_index++]);
+    }
+    else if(strcmp(argv[arg_index], "-MOC_src_tolerance") == 0) {
+      arg_index++;
+      RP._tolerance = atof(argv[arg_index++]);
+    }
+    else if(strcmp(argv[arg_index], "-max_iters") == 0) {
+      arg_index++;
+      RP._max_iters= atoi(argv[arg_index++]);
+    }
+    else if(strcmp(argv[arg_index], "-log_level") == 0) {
+      arg_index++;
+      RP._log_level = argv[arg_index++];
+    }
+    else if(strcmp(argv[arg_index], "-knearest") == 0) {
+      arg_index++;
+      RP._knearest = atoi(argv[arg_index++]);
+    }
+    else if(strcmp(argv[arg_index], "-CMFD_flux_update_on") == 0) {
+      arg_index++;
+      RP._CMFD_flux_update_on = atoi(argv[arg_index++]);
+    }
+    else if(strcmp(argv[arg_index], "-CMFD_centroid_update_on") == 0) {
+      arg_index++;
+      RP._CMFD_centroid_update_on = atoi(argv[arg_index++]);
+    } 
+    else if(strcmp(argv[arg_index], "-use_axial_interpolation") == 0) {
+      arg_index++;
+      RP._use_axial_interpolation = atoi(argv[arg_index++]);
+    } 
+    else if(strcmp(argv[arg_index], "-help") == 0) {
```

Comment:
Add --help and -h as other possibilities. We want the help function to be easily accessible

Reply:
added, good point